src/test/regress/expected/shard_rebalancer.out

```diff
@@ -1324,12 +1322,12 @@ SELECT * from master_set_node_property('localhost', :worker_2_port, 'shouldhaves
 
 (1 row)
 
-SELECT * FROM get_rebalance_table_shards_plan(threshold := 0, drain_only := true);
+SELECT * FROM get_rebalance_table_shards_plan(threshold := 0, drain_only := true) ORDER BY shardid;
```

Comment:
Why was this ORDER BY needed? normally get_rebalance_table_shards_plan should return a consistent order.

Reply:
Add comment that we're, sorting this plan because it involves different colocation groups and which colocation group is first moved is not consistent.